### PR TITLE
Update JSStackTrace.java

### PR DESCRIPTION
**Pull request by jamoreira**
Prevent exception "com.facebook.react.bridge.NoSuchKeyException: lineNumber" when start app:

`AndroidRuntime: FATAL EXCEPTION: mqt_native_modules
AndroidRuntime: com.facebook.react.bridge.NoSuchKeyException: lineNumber
AndroidRuntime:         at com.facebook.react.bridge.ReadableNativeMap.getValue(ReadableNativeMap.java:122)
AndroidRuntime:         at com.facebook.react.bridge.ReadableNativeMap.getInt(ReadableNativeMap.java:158)
AndroidRuntime:         at com.facebook.react.util.JSStackTrace.format(JSStackTrace.java:28)
AndroidRuntime:         at com.facebook.react.modules.core.ExceptionsManagerModule.reportSoftException(ExceptionsManagerModule.java:46)
AndroidRuntime:         at java.lang.reflect.Method.invoke(Native Method)
AndroidRuntime:         at com.facebook.react.bridge.JavaMethodWrapper.invoke(JavaMethodWrapper.java:372)
AndroidRuntime:        at com.facebook.react.bridge.JavaModuleWrapper.invoke(JavaModuleWrapper.java:160)
AndroidRuntime:         at com.facebook.react.bridge.queue.NativeRunnable.run(Native Method)
AndroidRuntime:        at android.os.Handler.handleCallback(Handler.java:790)
AndroidRuntime:         at android.os.Handler.dispatchMessage(Handler.java:99)
AndroidRuntime:        at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:29)
AndroidRuntime:         at android.os.Looper.loop(Looper.java:164)
AndroidRuntime:         at com.facebook.react.bridge.queue.MessageQueueThreadImpl$3.run(MessageQueueThreadImpl.java:192)
AndroidRuntime:        at java.lang.Thread.run(Thread.java:764)`

## Changelog

[CATEGORY] [TYPE] - Message

### CODE REVIEW

**Comment by cpojer** [comment]
cc @Salakar

File: ReactAndroid/src/main/java/com/facebook/react/util/JSStackTrace.java [hunk 1]
```diff
@@ -24,8 +24,17 @@ public static String format(String message, ReadableArray stack) {
       stringBuilder
         .append(frame.getString("methodName"))
         .append("@")
-        .append(stackFrameToModuleId(frame))
+        .append(parseFileId(frame))
         .append(frame.getInt("lineNumber"));
```
Comment (zhigang1992) [inline ReactAndroid/src/main/java/com/facebook/react/util/JSStackTrace.java]: Should this line be deleted?

**Comment by zhigang1992** [comment]
This should have already be fixed in https://github.com/facebook/react-native/pull/24399

**Comment by Salakar** [comment]
> cc @Salakar

Seems ok to me @cpojer - but this has been fixed on latest, looks like this PR is targetting 0.55

**Comment by cpojer** [comment]
Oh yeah I missed that too. Sorry, we are not merging PRs into very old releases.

**Pull request closed without merging**